language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/traces/LogDTO.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ballerinalang.langserver.extensions.ballerina.traces;
+
+/**
+ * {@link LogDTO} Parsed log message sent to client.
+ *
+ */
+public class LogDTO {
```

Comment:
dont use DTO suffix

Reply:
rename to TraceLog

Reply:
this is to be consistant with other models used in Json RPC - they don't use DTO suffix for naming dtos